src/ConnectedMode/Migration/MSBuildFileCleaner.cs

```diff
@@ -63,7 +63,7 @@ public string Clean(string content, LegacySettings legacySettings, CancellationT
 
             if (!nodesToRemove.Any())
             {
-                logger.LogVerbose("No settings to remove");
+                logger.LogVerbose("    No settings to remove");
```

Comment:
Cosmetic: a non-verbose message about the file being changed is written just before this message. Indenting this verbose message makes it easier to see the two messages are related.